packages/main/src/themes/Toolbar.css

```diff
@@ -0,0 +1,122 @@
+:host(:not([hidden])) {
+	display: inline-block;
+	width: 100%;
+	height: 2.75rem;
+	background: var(--ui5-toolbar-Background);
+	border-bottom: 0.0625rem solid var(--sapGroup_ContentBorderColor);
+	flex-shrink: 1;
+	color: var(--sapButton_TextColor);
+}
+
+:host(:not([hidden])[data-ui5-compact-size]) {
+	height: 2rem;
+}
+
+:host([disabled]) {
+	opacity: .5;
+	pointer-events: none;
+}
+
+:host([design="Solid"]) {
+	background: var(--sapBaseColor);
+}
+
+:host([design="Transparent"]) {
+	background: var(--sapToolbar_Background);
+}
+
+:host([styling="Clear"]) {
+	border-bottom: 0;
+}
+
+.ui5-tb-root {
+	/*min-width: 2.5rem;*/
+	width: 100%;
+	padding: 0 0.5rem;
+	justify-content: flex-end;
+	align-items: center;
+	overflow: hidden;
+	pointer-events: none;
+}
+
+.ui5-tb-root > * {
+	pointer-events: all;
+}
+
+:host([no-padding]) .ui5-tb-root {
+	padding: 0;
+}
+
+:host([align-content="End"]) .ui5-tb-root {
+	justify-content: flex-end;
+}
+
+:host([align-content="Start"]) .ui5-tb-root {
+	justify-content: flex-start;
+}
+
+.ui5-tb-root,
+.ui5-tb-actions {
+	display: flex;
+	height: 100%;
+	white-space: nowrap;
+	align-items: center;
+	outline: none;
+	box-sizing: border-box;
+}
+
+.ui5-tb-actions-full-width {
+    width: 100%;
+}
+
+.ui5-tb-start {
+	display: flex;
+	flex-shrink: 0;
+}
+
+.ui5-tb-actions {
+	justify-content: flex-end;
+}
+
+.ui5-tb-item[design="Transparent"] {
+	color: inherit;
+}
+
+
+.ui5-tb-item {
+    flex-shrink: 0;
+}
+
+.ui5-tb-item:not(:last-child) {
+	margin-right: 0.25rem;
+}
+
+.ui5-tb-spacer {
+    min-width: 1rem;
+}
+
+.ui5-tb-separator {
+    height: 2rem;
+	width: 0.0625rem;
+	height: 2rem;
+	background: var(--sapToolbar_SeparatorColor);
+	box-sizing: border-box;
+}
+
+.ui5-tb-separator[data-ui5-compact-size] {
+	height: 1.5rem;
+}
+
+.ui5-tb-action-content {
```

Comment:
the class `ui5-tb-action-content` is never set in the dom as well, perpahs the styles below are not needed?